.github/workflows/ci.yml

```diff
@@ -342,9 +344,9 @@ jobs:
         - name: linux_nvcc-11.3_gcc-8_release
           os: ubuntu-latest
           env: { CXX: g++,     CC: gcc,    ALPAKA_CI_GCC_VER: 8,        ALPAKA_CI_STDLIB: libstdc++, CMAKE_BUILD_TYPE: Release, ALPAKA_CI_BOOST_BRANCH: boost-1.69.0, ALPAKA_CI_CMAKE_VER: 3.18.6,                     ALPAKA_CI_DOCKER_BASE_IMAGE_NAME: "ubuntu:18.04", ALPAKA_ACC_GPU_CUDA_ENABLE: ON, ALPAKA_CI_CUDA_VERSION: "11.3", CMAKE_CUDA_COMPILER: nvcc,                                              ALPAKA_ACC_CPU_B_SEQ_T_FIBERS_ENABLE: OFF }
-        - name: linux_nvcc-11.3_gcc-9_release
+        - name: linux_nvcc-11.3_gcc-9_debug
           os: ubuntu-latest
-          env: { CXX: g++,     CC: gcc,    ALPAKA_CI_GCC_VER: 9,        ALPAKA_CI_STDLIB: libstdc++, CMAKE_BUILD_TYPE: Release, ALPAKA_CI_BOOST_BRANCH: boost-1.69.0, ALPAKA_CI_CMAKE_VER: 3.19.7,                     ALPAKA_CI_DOCKER_BASE_IMAGE_NAME: "ubuntu:18.04", ALPAKA_ACC_GPU_CUDA_ENABLE: ON, ALPAKA_CI_CUDA_VERSION: "11.3", CMAKE_CUDA_COMPILER: nvcc,                                              ALPAKA_ACC_CPU_B_SEQ_T_FIBERS_ENABLE: OFF }
+          env: { CXX: g++,     CC: gcc,    ALPAKA_CI_GCC_VER: 9,        ALPAKA_CI_STDLIB: libstdc++, CMAKE_BUILD_TYPE: Debug,   ALPAKA_CI_BOOST_BRANCH: boost-1.69.0, ALPAKA_CI_CMAKE_VER: 3.19.7,                     ALPAKA_CI_DOCKER_BASE_IMAGE_NAME: "ubuntu:18.04", ALPAKA_ACC_GPU_CUDA_ENABLE: ON, ALPAKA_CI_CUDA_VERSION: "11.3", CMAKE_CUDA_COMPILER: nvcc,                                              ALPAKA_ACC_CPU_B_SEQ_T_FIBERS_ENABLE: OFF }
```

Comment:
All other nvcc/g++ builds are release, so I changed one to debug to also test that. I think I wanted to try something because of the failing AVX512 intrinsics in release mode. It is unrelated to adding TBB, but I would keep it.